core/modules/file.py

```diff
@@ -128,6 +131,98 @@ def thumbnailer(fileSkel, existingFiles, params):
     return resList
 
 
+def cloudfunction_thumbnailer(fileSkel, existingFiles, params):
+    if not conf.get("viur.file.thumbnailerURL", False):
+        raise ValueError("viur.file.thumbnailerURL is not set")
+
+    import requests as _requests
+    orgifileName = fileSkel["name"].replace("&#040;", "(").replace("&#041;", ")").replace("&#061;", "=")
+    if utils.isLocalDevelopmentServer:
+        signedUrl = utils.downloadUrlFor(fileSkel["dlkey"], fileSkel["name"])
+    else:
+        blob = bucket.get_blob("%s/source/%s" % (fileSkel["dlkey"],orgifileName))
+        if not blob:
+            logging.warning("Blob %s/source/%s is missing from Cloudstore!" %  (fileSkel["dlkey"],orgifileName))
+            return
+
+        authRequest = requests.Request()
+        expiresAt = datetime.now() + timedelta(seconds=60)
+        signing_credentials = compute_engine.IDTokenCredentials(authRequest, "")
+        contentDisposition = "filename=%s" % fileSkel["name"]
+        signedUrl = blob.generate_signed_url(
+            expiresAt,
+            credentials=signing_credentials,
+            response_disposition=contentDisposition,
+            version="v4")
+
+    dataDict = {"url": signedUrl,
+                "name": fileSkel["name"],
+                "params": params,
+                "minetype": fileSkel["mimetype"],
+                "baseUrl": utils.currentRequest.get().request.host_url.lower(),
+                "targetKey": fileSkel["dlkey"],
+                "nameOnly": True
+                }
```

Comment:
Please follow PEP8-guideline
```suggestion
    dataDict = {
        "url": signedUrl,
        "name": fileSkel["name"],
        "params": params,
        "minetype": fileSkel["mimetype"],
        "baseUrl": utils.currentRequest.get().request.host_url.lower(),
        "targetKey": fileSkel["dlkey"],
        "nameOnly": True
    }
```

Reply:
there is a typo here:
`"minetype": fileSkel["mimetype"],
`
needs to be

`"mimetype": fileSkel["mimetype"],`